.github/workflows/newdeploy.yml

```
@@ -0,0 +1,353 @@
+name: Build and Push Docker Image to ECR
+
+
+on:
+  push:
+    branches:
+      - feature/3.1.1 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)